From 20a51f6e5ebc56707554a52e2fb6a61bf6511315 Mon Sep 17 00:00:00 2001
From: Markus Armbruster <armbru@redhat.com>
Date: Thu, 21 Feb 2019 09:11:00 +0000
Subject: [PATCH 1/2] fdc: Revert downstream disablement of device "floppy"

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <20190221091101.31999-2-armbru@redhat.com>
Patchwork-id: 84691
O-Subject: [RHEL-8.0/AV qemu-kvm PATCH v2 1/2] fdc: Revert downstream disablement of device "floppy"
Bugzilla: 1664997
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>

Board code creates floppy devices on behalf of -drive if=floppy,...
When they got qdevifified, they also became available with -device.
We made it unavailable downstream as per our policy to permit new
devices only when we have a use for them (commit 0533a6ee98f).  We now
have a use: we need it to move from -drive to -blockdev.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 hw/block/fdc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/hw/block/fdc.c b/hw/block/fdc.c
index 56b7aeb..6f19f12 100644
--- a/hw/block/fdc.c
+++ b/hw/block/fdc.c
@@ -599,7 +599,6 @@ static void floppy_drive_class_init(ObjectClass *klass, void *data)
     k->bus_type = TYPE_FLOPPY_BUS;
     k->props = floppy_drive_properties;
     k->desc = "virtual floppy drive";
-    k->user_creatable = false; /* RH state preserve */
 }
 
 static const TypeInfo floppy_drive_info = {
-- 
1.8.3.1

